src/App.js

```diff
@@ -87,6 +89,26 @@ export default class App extends Component {
     }
   }
 
+  /* * * * * * * * * * * * * * * * *
+   *
+   * SET KONAMI EASTER EGG
+   *
+   * * * * * * * * * * * * * * * * */
+  setKonami () {
+    if (this.props.meta.konami === '') return ;
+    const konami = '38,38,40,40,37,39,37,39,66,65'
+    const keys = []
```

Comment:
const keys ---> this.state.keystrokes_history (et ajouter un shouldComponentUpdate quelquepart, parce que pas besoin de re-render l'app si ce truc change)